src/PrestaShopBundle/Controller/Admin/FrameworkBundleAdminController.php

```diff
@@ -80,11 +84,19 @@ public function overviewAction()
         @trigger_error(__FUNCTION__ . 'is deprecated since version 8.0 and will be removed in the next major version.', E_USER_DEPRECATED);
 
         return [
-            'is_shop_context' => (new Context())->isShopContext(),
+            'is_shop_context' => $this->get('prestashop.adapter.shop.context')->isShopContext(),
             'layoutTitle' => empty($this->layoutTitle) ? '' : $this->trans($this->layoutTitle, 'Admin.Navigation.Menu'),
         ];
     }
 
+    /**
+     * @return ShopConfigurationInterface
+     */
+    protected function getConfiguration(): ShopConfigurationInterface
+    {
+        return $this->container->get('prestashop.adapter.legacy.configuration');
```

Comment:
```suggestion
        return $this->container->get(ShopConfigurationInterface::class);
```
Shouldn't we use the FQCN way instead of the named service?